### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**

### CODE REVIEW

**Review by AniaHna — COMMENTED** [review]
Good job!
 😉

File: src/arrayMethodJoin.js [hunk 1]
```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separ = ',') {
+    let result = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const element = this[i];
```
Comment (silchyn) [inline src/arrayMethodJoin.js:11]: Why we need this const?

**Review by silchyn — CHANGES_REQUESTED** [review]

File: src/arrayMethodJoin.js (hunk 1 above)
Comment (roman-zherebetskyi) [reply src/arrayMethodJoin.js:11]: 'element' sounds better :grin:
okay I'll change, my bad...

**Review by silchyn — APPROVED** [review]
Cool!

**Pull request closed without merging**